addons/calendar/models/calendar_event.py

```diff
@@ -531,6 +531,13 @@ def create(self, vals_list):
 
         for vals in recurring_vals:
             vals['follow_recurrence'] = True
+            # When updating time values of recurrence, set attendees state as 'needAction' (except for organizer).
+            if self.env.context.get('update_time'):
+                organizer = self.env['res.users'].browse(vals['user_id'])
+                attendee_ids = vals.get('attendee_ids', [])
+                for attendee in attendee_ids:
+                    if len(attendee) == 3 and attendee[2].get('partner_id') != organizer.partner_id.id:
+                        attendee[2]['state'] = 'needsAction'
```

Comment:
Not a big fan of relying on the commands, but I don't see a better way of dealing with this